src/CorsMiddleware.php

```diff
@@ -325,10 +333,10 @@ private function cache(int $cache): void
     private function error(callable $error): void
     {
         if ($error instanceof Closure) {
-            $this->options["error"] = $error->bindTo($this);
-        } else {
-            $this->options["error"] = $error;
+            $error->bindTo($this);
         }
+
+        $this->options["error"] = $error;
     }
```

Comment:
Does rector do this? AFAIK this changes behaviour. According to docs `$error->bindTo()` returns a new anonymous function bound to a new scope. So before in case of Closure the error handler was bound to `$this`. Now error handler is always the original which is not bound to `$this`.

Reply:
It seems there was no test for this. Added now: https://github.com/tuupola/cors-middleware/pull/66

Reply:
Ouh yeah, that was my fault. :sweat_smile: 
Thanks for adding the test, I also added one for the `methods` callable.